cloudify_cli/commands/cluster.py

```diff
@@ -100,8 +100,14 @@ def _inner(client, *args, **kwargs):
 
 def _all_in_one_manager(client):
     manager_nodes = client.manager.get_managers().items
+    if len(manager_nodes) > 1:
```

Comment:
this is not correct check, all of them need to be one

Reply:
but it suffices that we have more than 1 of any of the node types to determine it's not an AIO

Reply:
Well, there is not true, because you can have a cluster with one manager  also for the other but not together